confirm.c

```diff
@@ -781,18 +785,65 @@ confirm_query_create6(const struct sockaddr_storage *dst, uint8_t ttl,
 	query->traffic_class = traffic_class;
 	query->flow_label = flow_label;
 	query->icmpid = icmpid;
+	query->flowid = flowid & CONFIRM_MAX_FLOWID;
+	query->padding = 0;
+	query->revflow = 0;
+	query->ntries = 3;
```

Comment:
this seems to be repeated in both `create_icmp` and `create_tcp`. Should we gather the defaults in a `create_defaults` and avoid the duplicated code?